.travis.yml

```diff
@@ -26,6 +26,8 @@ before_install:
 - source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
 before_script:
 - source ${TRAVIS_BUILD_DIR}/tools/ci/before_script.sh
+after_script:
+- source ${TRAVIS_BUILD_DIR}/tools/ci/after_script.sh
```

Comment:
Minor, but can you move this physically after the script section?  It makes it clearer on the order of execution.